rescript/get_unite.py

```diff
@@ -10,10 +10,13 @@
 import tempfile
 import tarfile
 import requests
-from requests.exceptions import HTTPError
 
 from pandas import DataFrame
-from q2_types.feature_data import TaxonomyFormat, DNAFASTAFormat, DNAIterator
+from q2_types.feature_data import (
+    TaxonomyFormat,
+    MixedCaseDNAFASTAFormat,
+    DNAIterator,
+)
```

Comment:
@mikerobeson This would explain why your testing showed no slowdown!

Reply:
Haha right! I'll do a quick download and run. If that all works I'll just go ahead and merge.